universal/ubi8/Dockerfile

```
@@ -440,6 +434,20 @@ cd -
 rm -rf "${TEMP_DIR}"
 EOF
 
+# Bash completions
+RUN dnf -y install bash-completion
```

In general, it's undesirable to have multiple `dnf install` steps in a dockerfile as it can explode the image size (I don't know if this image does anything to mitigate this, so this comment may be useless). In any case, each dnf install should be accompanied with clean up of the cache, etc. 

```suggestion
RUN dnf -y install bash-completion
    && dnf clean all \
  	&& rm -rf /var/cache/yum
```

---

I think the dnf clean already happened a few lines below.